src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -13357,6 +13373,31 @@ private bool IsInQuery
             bool allowTrailingSeparator,
             bool requireOneElement,
             bool allowSemicolonAsSeparator) where TNode : GreenNode
+        {
+            return ParseCommaSeparatedSyntaxList(
+                ref openToken,
+                closeTokenKind,
+                isPossibleElement,
+                parseElement,
+                immediatelyAbort: null,
+                skipBadTokens,
+                allowTrailingSeparator,
+                requireOneElement,
+                allowSemicolonAsSeparator);
+        }
+
+#nullable enable
```

Comment:
Why not enable the above method?

Reply:
this was hte method where i added something nullable :)